devdoc/worker_thread_requirements.md

```diff
@@ -0,0 +1,146 @@
+﻿`worker_thread` requirements
+================
+
+## Overview
+
+`worker_thread` is a helper module that implements execution of a work item on a separate thread. It can be used in batching operations, sequencing completions, etc.
+The user instantiates a BS worker thread instance and then signals when the work shall be executed by calling `worker_thread_schedule_process`.
```

Comment:
>BS [](http://example.com/codeflow?start=24&length=2)

Can you please remove the BS here and just have it as:

"a worker thread ..."

Reply:
Removed BS